src/plugin.js

```diff
@@ -451,7 +452,7 @@ class VR extends Plugin {
     this.player_.addChild('BigVrPlayButton', {}, this.bigPlayButtonIndex_);
     this.player_.bigPlayButton = this.player_.getChild('BigVrPlayButton');
     // mobile devices
-    if (videojs.browser.IS_ANDROID || videojs.browser.IS_IOS) {
+    if (videojs.browser.IS_ANDROID || videojs.browser.IS_IOS || this.options_.forceCardboard) {
```

Comment:
in here, do we know if we're using native VR or not? Would be good to have it added for the firefox usecase.